azkaban-common/src/main/java/azkaban/user/UserUtils.java

```diff
@@ -78,12 +78,12 @@ static void setupWatch(final Map<String, ParseConfigFile> configFileMap) throws
       }
 
       try {
-        Path dir = Paths.get(fileName).getParent();
+        final Path dir = Paths.get(fileName).getParent();
         if (!dirToFilesMap.containsKey(dir)) {
-          // There is not entry for this directory, create a watchkey
+          // There is no entry for this directory, create a watchkey
           WatchKey watchKey = dir.register(watchService,
               new WatchEvent.Kind[]{StandardWatchEventKinds.ENTRY_MODIFY},
-              SensitivityWatchEventModifier.HIGH);
```

Comment:
It seems like reducing the sensitivity would make the test failure less likely to occur, but not fix the issue?